### DISCUSSION TITLE
FP 1.12.4 - How to reshape the menus space occupation?

### DISCUSSION THREAD

**Discussion by Illegit** [freeplane/freeplane#1994] [Help] [answered]
Hi, I have a problem with the screen space occupation by the menus, I think I miss 2 lines space in both of my laptops I use daily
![png-240824-114301665-1223473124522042398](https://github.com/user-attachments/assets/ce7646fa-f532-4b48-8234-acb36a00fd48)
1) I understood that one way is to reduce/change the size of the writings/characters and so on, but I already have a tradeoff between reading and space, eventually in this case which parameters to re-check?
2) now another possibility is using the tool panel and applying the Edo tools to it (but menubar, toolbar, filter toolbar are not fully movable: MoM applies only to the commands)
3) or anything else?
Thanks

**Comment by dpolivaev** [comment, marked as answer]
Reduce Preferences…->Appearance->Look and feel->Toolbar icon height

**Reply by Illegit** [reply]
Thanks. I changed fro 10 points to 8.7 points and now it is correct.
